component/core/src/main/java/org/exoplatform/social/core/upgrade/SpaceNodeIconsPlugin.java

```diff
@@ -0,0 +1,84 @@
+package org.exoplatform.social.core.upgrade;
```

Comment:
Please move class to `io.meeds`. Any new class has to use this package name until we upgrade all existing classes to this new package name.

```suggestion
package io.meeds.social.core.upgrade;
```

Reply:
fixed